.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ references:
   #        Dependency Anchors
   # -------------------------
   dependency_versions:
-    xcode_version: &xcode_version "14.0.1"
+    xcode_version: &xcode_version "14.3.0"
```

Comment:
I will not change the Version of Xcode to fix this in 0.71 now as it can require some other changes in the CricleCI configuration, which is red right now.